README.md

```diff
@@ -643,6 +643,7 @@ Table of Contents
   * [unpkg.com](https://unpkg.com/) — CDN for everything on npm
   * [Web Support Revolution](https://w.tools/) — Free CDN, backup, firewall, antivirus and monitoring.
   * [Namecheap Supersonic](https://www.namecheap.com/supersonic-cdn/#free-plan) — Free DDoS protection
+  * [BotFront](https://botfront.net/#pricing) — Free CDN and DDoS protection
```

Comment:
pls add something like 'for one domain' at the end and update the link to go just to the main url not `#pricing`

thanks!

Reply:
Yeah, sure. Done.